hax/__init__.py

```diff
@@ -1 +1,63 @@
-from . import config, ipython, looproot, minitrees, runs, utils, waveforms
\ No newline at end of file
+import logging
+import os
+import inspect
+from configparser import ConfigParser
+import socket
+
+__version__ = 0.1
+
+# Stitch the package together
+# I am surprised this works (even if we do 'from hax' instead of 'from .')
+# as some of the modules do 'import hax' or 'from hax.foo import bar'.. shouldn't we get circular imports??
+# I need to read up on python packaging more...
```

Comment:
__[flake8]__
* [E402] module level import not at top of file
* [F401] 'ipython' imported but unused
* [F401] 'minitrees' imported but unused
* [F401] 'paxroot' imported but unused
* [F401] 'pmt_plot' imported but unused
* [F401] 'raw_data' imported but unused
* [F401] 'runs' imported but unused
* [F401] 'utils' imported but unused